SPECS-EXTENDED/kernel-rt/kernel-rt.spec

```diff
@@ -121,11 +126,13 @@ Requires:       audit
 %description tools
 This package contains the 'perf' performance analysis tools for Linux kernel.
 
-%package -n     python3-perf
+%package -n     python3-perf-rt
 Summary:        Python 3 extension for perf tools
 Requires:       python3
+Requires:       %{name} = %{version}-%{release}
+Provides:       python3-perf
 
-%description -n python3-perf
+%description -n python3-perf-rt
```

Comment:
My concerns with this approach:
- These two packages seem to have no run-time dependency on any kernel binaries. Or at least nothing, that RPM could detect automatically - running `dnf repoquery --requires --resolve --recursive bpftool` returns no kernel subpackages. It seems to me, that it's very likely, they don't really have a dependency on any specific kernel version of build. Or in other words: these tools seem no different from tools produced by other specs.
- Our build tools are not smart enough to pick the right version of these packages when building an image. The tools check one dependency at a time, so a dependency on `python3-perf` would present multiple candidates. The candidates resolution mechanism (we ask RPM to pick the package it would prefer) is where we're going to probably end up picking packages at random, because at that point in the build it's unknown, which kernel we're going to need.
- I feel like we're introducing unnecessary redundancies with these two packages.

Considering that, I think we might be better off building them only from one kernel spec and removing them from all other ones.